src/main/java/seedu/duke/parsers/Parser.java

```diff
@@ -66,9 +68,30 @@ public Parser() {
         if (groupNames.contains(INVALID)) {
             String invalidInput = parsedCommand.get(INVALID);
             if (!Objects.isNull(invalidInput) && !invalidInput.isBlank()) {
-                throw new InvalidInputException(invalidInput);
+                throw new InvalidExcessArgumentException(invalidInput);
```

Comment:
I think this exceeds 30 LOC based on the guidelines, so consider extracting them into another function

Reply:
Yup you are right, extracted.